distribution/lib/Standard/Database/0.0.0-dev/src/Data/Table.enso

```diff
@@ -165,10 +165,6 @@ type Table
            `Missing_Input_Columns` is raised as an error, unless
            `error_on_missing_columns` is set to `False`, in which case the
            problem is reported according to the `on_problems` setting.
-         - If a column index is out of range, a `Column_Indexes_Out_Of_Range` is
-           raised as an error, unless `error_on_missing_columns` is set to
-           `False`, in which case the problem is reported according to the
-           `on_problems` setting.
```

Comment:
I was mostly removing these without replacement, because IMO the upper comment is good enough for both index and name based scenarios:

```
- If a column in `columns` is not in the input table, a
  `Missing_Input_Columns` is raised
```

If a "column" is not in the input table IMO works both when the column is represented by name, selector or the integer index.